src/main/java/cz/cvut/kbss/study/service/PatientRecordService.java

```diff
@@ -87,4 +88,19 @@ public interface PatientRecordService extends BaseService<PatientRecord> {
      * @return List of matching records
      */
     Page<RawRecord> exportRecords(RecordFilterParams filters, Pageable pageSpec);
+
+
+
+    /**
+     * Retrieves a set of all distinct phases that records can be in.
+     * <p>
+     * This method provides a way to get a comprehensive list of the various phases
+     * associated with records. Phases typically represent different stages or statuses
+     * that a record might go through in its lifecycle.
+     *
+     * @return Set of all available record phases
```

Comment:
neni available zavadijici?

If you will have no records you will have zero "available" phases. It is rather about phases being used in records, how about renaming?:

available --> used